src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -6854,5 +6854,52 @@ public R M4()
 ").VerifyDiagnostics(
                 );
         }
+
+        [Fact]
+        [WorkItem(39960, "https://github.com/dotnet/roslyn/issues/39960")]
+        public void MissingExceptionType()
```

Comment:
>MissingExceptionType [](start = 20, length = 20)

Please add a regression test verifying that we are able to compile the code below with System.Exception missing at compile time and execute the code successfully:

```
class C
{
    static void Main()
    {
        try
        {
            Test();
        }
        catch
        {
            System.Console.WriteLine("in catch");
        }
    }

    static void Test()
    {
        throw null;
    }
}
```

Reply:
For 7.0 at least.

---
In reply to: [359067146](https://github.com/dotnet/roslyn/pull/40145#discussion_r359067146) [](ancestors = 359067146)

Reply:
Sure, though that would not exercise any code in this PR.